src/utils/navigation.ts

```diff
@@ -245,6 +244,16 @@ export const NAVIGATION: Navigation = {
     previousPage: () => config.UPDATE_BENEFICIAL_OWNER_TYPE_URL,
     nextPage: [config.UPDATE_BENEFICIAL_OWNER_TYPE_URL]
   },
+  [config.UPDATE_REVIEW_BENEFICIAL_OWNER_INDIVIDUAL_URL]: {
```

Comment:
do we ever expect it to be called without index param?

Reply:
No, this is the part that is similar to the 2 redirects on the page. Both lines 69 and 83 use it, params is added on each on the lines